multimedia/zvbi/Portfile

```diff
@@ -53,9 +53,9 @@ depends_lib-append \
                     port:libiconv \
                     port:libpng
 
-# Disable rpath use for 10.5 and earlier
 # See: https://trac.macports.org/ticket/64212
-if { ${os.platform} eq "darwin" && ${os.major} <= 9 } {
+# Also: https://trac.macports.org/ticket/70160
+if {[string match *gcc* ${configure.compiler}]} {
```

Comment:
You don't want to disable it on all systems?

Reply:
Yeah, let me do it, thanks